icicle/include/icicle/hash/keccak.h

```diff
@@ -15,7 +15,7 @@ namespace icicle {
    */
   Hash create_keccak_256_hash(uint64_t input_chunk_size = 0);
   struct Keccak256 {
-    inline static Hash create(uint64_t input_chunk_size = 0) { return create_keccak_256_hash(); }
+    inline static Hash create(uint64_t input_chunk_size = 0) { return create_keccak_256_hash(input_chunk_size); }
```

Comment:
maybe better rename it default_input_chunk_size?

Reply:
done